components/script/dom/servoparser/mod.rs

```diff
@@ -648,13 +648,11 @@ impl FetchResponseListener for ParserContext {
                     parser.parse_sync();
                 }
             },
-            Some(ContentType(Mime(TopLevel::Text, SubLevel::Xml, _))) => {}, // Handle text/xml
+            // Handle xml (text/xml, application/xml, application/xhtml+xml)
+            Some(ContentType(Mime(TopLevel::Text, ref sublevel @ _, _)))
```

Comment:
This means that we also accept text/xhtml+xml, which is not a MIME type that we want to support.